Game/world/npc.cpp

```diff
@@ -3088,7 +3088,9 @@ SensesBit Npc::canSenseNpc(float tx, float ty, float tz, bool freeLos, float ext
   if(!freeLos){
     float dx  = x-tx, dz=z-tz;
     float dir = angleDir(dx,dz);
-    float da  = float(M_PI)*(angle-dir)/180.f;
+    float boneRotation = rotation(); //Fallback is NPC rotation.
+    (void)visual.boneRotationY("BIP01", boneRotation);
```

Comment:
This line look confusing, as suggested above `boneRotationY` can perform fallback by itself, so return code is not needed.